src/NosCore.GameObject/Providers/InventoryService/InventoryService.cs

```diff
@@ -350,6 +351,15 @@ public InventoryItemInstance DeleteFromTypeAndSlot(NoscorePocketType type, short
                 return false;
             }
 
+            // check if the item is a palced MinilandObject
+            if (sourcetype == NoscorePocketType.Miniland)
+            {
+                if (sourcePocket.ItemInstance is MinilandObjectDto || destinationPocket.ItemInstance is MinilandObjectDto)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Merge this if statement with the enclosing one.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=4971874)